nrf_rpc/doc/transport.rst

```diff
@@ -0,0 +1,64 @@
+.. _transport:
+
+Transport
+#########
+
+Bottom layer of the nRF_RPC is a transport layer.
+It is OS-dependent.
+Its main goal is to transfer packets between two sites and to handle thread pool.
```

Comment:
What is "sites"?  Cores or Threads?

Reply:
nRF RPC is for inter-core communication, so *sides* are always different cores.

Reply:
So, it has to be _sides_ in the doc?

Reply:
Yes, `sides`, I will fix that.